javascript/react-verbs/src/common/hooks.ts

```diff
@@ -40,15 +46,21 @@ export function useHandleDropdownChange(
 	)
 }
 
+// #endregion
+
+// #region Textfield Change Handler
+
+export type TextfieldChangeHandler = (
```

Comment:
This is super persnickety, but fluent caps the component as 'TextField' so it might be a little more clear